### PR TITLE
Add PageHeader docs

### PR DESCRIPTION
**Pull request by taysea**
Add docs for PageHeader

### CODE REVIEW

File: src/screens/PageHeader.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        },
+      ]}
+      description="A paragraph of text"
+      intrinsicElement="p"
+      code={`<PageHeader
+      title="Page Title"
+      subtitle="A subtitle for the page."
+      parent={<Anchor label="Parent Page" />}
+      actions={<Button label="Edit" primary />}
+    />`}
```
Comment (jcfilben) [inline src/screens/PageHeader.js:53]: Remove extra spacing in code example
```suggestion
      code={`<PageHeader
    title="Page Title"
    subtitle="A subtitle for the page."
    parent={<Anchor label="Parent Page" />}
    actions={<Button label="Edit" primary />}
/>`}
```

File: src/screens/PageHeader.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+        <Property name="children">
+          <Description>
+            Any content to be rendered directly beneath the PageHeader.
+          </Description>
+          <PropertyValue type="element">
+            <Example>{`<NameValueList>
+    <NameValuePair name="Created On">Jan 10, 2022</NameValuePair>
+    <NameValuePair name="Last Edited">August 12, 2022</NameValuePair>
+</NameValueList>`}</Example>
```
Comment (jcfilben) [inline src/screens/PageHeader.js:80]: Adjusted spacing to make it easier to read this example
```suggestion
            <Example>{`<NameValueList>
    <NameValuePair name="Created On">
        Jan 10, 2022
    </NameValuePair>
    <NameValuePair name="Last Edited">
        August 12, 2022
    </NameValuePair>
</NameValueList>`}</Example>
```

File: src/screens/PageHeader.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+        </Property>
+
+        <Property name="pageHeader.subtitle">
+          <Description>Any valid Paragraph props for the subtitle.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`{
+fill: true,
+margin: 'none'
+}`}
```
Comment (jcfilben) [inline src/screens/PageHeader.js:178]:
```suggestion
              {`{
  fill: true,
  margin: 'none'
}`}
```

File: src/screens/PageHeader.js [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+        </Property>
+
+        <Property name="pageHeader.title">
+          <Description>Any valid Heading props for the title.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`{
+fill: true,
+margin: 'none',
+}`}
```
Comment (jcfilben) [inline src/screens/PageHeader.js:190]:
```suggestion
              {`{
  fill: true,
  margin: 'none',
}`}
```

File: src/screens/PageHeader.js [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+small: {
+    areas: [
+      ['parent', 'null'],
+      ['title', 'actions'],
+      ['subtitle', 'actions'],
+    ],
+    columns: [['small', 'flex'], 'auto'],
+    rows: ['auto', 'auto', 'auto'],
+    gap: { row: 'xsmall', column: 'large' },
+  },
```
Comment (jcfilben) [inline src/screens/PageHeader.js:212]:
```suggestion
},
```

File: src/screens/PageHeader.js [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+medium: {
+    areas: [
+      ['parent', 'null'],
+      ['title', 'actions'],
+      ['subtitle', 'actions'],
+    ],
+    columns: [['medium', 'large'], 'auto'],
+    rows: ['auto', 'auto', 'auto'],
+    gap: { row: 'xsmall', column: 'medium' },
+  },
```
Comment (jcfilben) [inline src/screens/PageHeader.js:235]:
```suggestion
},
```

File: src/screens/PageHeader.js [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+large: {
+    areas: [
+      ['parent', 'null'],
+      ['title', 'actions'],
+      ['subtitle', 'actions'],
+    ],
+    columns: [['medium', 'large'], 'auto'],
+    rows: ['auto', 'auto', 'auto'],
+    gap: { row: 'xsmall', column: 'large' },
+  },
```
Comment (jcfilben) [inline src/screens/PageHeader.js:258]:
```suggestion
},
```

File: src/screens/PageHeader.js [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+
+const extendedGenericSyntaxes = JSON.parse(JSON.stringify(genericSyntaxes));
+extendedGenericSyntaxes.margin.unshift('none');
```
Comment (jcfilben) [inline src/screens/PageHeader.js:23]: What are we using this part for?

File: src/screens/PageHeader.js [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+
+const PageHeaderPage = () => (
+  <Page>
+    <ComponentDoc
+      name="PageHeader"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Type-Paragraph&full=0&stories=1&panelRight=0',
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
```
Comment (jcfilben) [inline src/screens/PageHeader.js:33]: This is taking us to the Paragraph stories instead of PageHeader

File: src/screens/PageHeader.js [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+          label: 'Storybook',
+        },
+        {
+          url: 'https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=/paragraph&module=%2Fsrc%2FParagraph.js',
```
Comment (jcfilben) [inline src/screens/PageHeader.js:37]: This is linking to the Paragraph codesandbox. If we don't have a codesandbox for PageHeader we can remove this

File: src/screens/PageHeader.js [hunk 11]
```diff
@@ -0,0 +1,281 @@
...
+          badge: 'https://codesandbox.io/static/img/play-codesandbox.svg',
+          label: 'CodeSandbox',
+        },
+        {
+          url: 'https://github.com/grommet/grommet/tree/master/src/js/components/Paragraph',
+          label: 'Github',
```
Comment (jcfilben) [inline src/screens/PageHeader.js:43]: This should be updated to link to the PageHeader file on Grommet's Github

File: src/screens/PageHeader.js [hunk 12]
```diff
@@ -0,0 +1,281 @@
...
+        },
+      ]}
+      description="A paragraph of text"
+      intrinsicElement="p"
```
Comment (jcfilben) [inline src/screens/PageHeader.js:47]: These need updated

File: src/screens/PageHeader.js (hunk 8 above)
Comment (taysea) [reply src/screens/PageHeader.js]: Done.

File: src/screens/PageHeader.js (hunk 9 above)
Comment (taysea) [reply src/screens/PageHeader.js]: Fixed.

File: src/screens/PageHeader.js (hunk 10 above)
Comment (taysea) [reply src/screens/PageHeader.js]: Done.

File: src/screens/PageHeader.js (hunk 11 above)
Comment (taysea) [reply src/screens/PageHeader.js]: Done.

File: src/screens/PageHeader.js (hunk 12 above)
Comment (taysea) [reply src/screens/PageHeader.js]: Done, updated to `header` since that is the intrinsic element

**Review by jcfilben — APPROVED** [review]
Looks good!

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**